### ISSUE TITLE
Module not found: Error: Can't resolve 'clsx'

### ISSUE THREAD

**Issue by QuesoCaliente** [pheralb/superkey#16]
I think the new version does not work, you could check it, I also tried to manually apply the changes regarding the problem of not being able to click on the sides of the modal but it is not a valid solution either, it did not change nothing, the solution I initially offered could be applied to the component Dialog and also fix it

![image](https://user-images.githubusercontent.com/43506340/216172192-a6a6e016-3b53-4fee-96af-19f2ba03da4d.png)

**Comment by pheralb** [comment]
Thanks for the report, I will fix it 🚀.

**Comment by pheralb** [comment]
🎉 Fixed in [v0.2.2](https://github.com/pheralb/superkey/releases/tag/v0.2.2).

- ✅ Module not found: Error: Can't resolve 'clsx' when building to production:

Using **Next.js 13 + /pages**:

```
> next build

info  - Linting and checking validity of types
info  - Creating an optimized production build
info  - Compiled successfully
info  - Collecting page data
info  - Generating static pages (3/3)
info  - Finalizing page optimization

Route (pages)                              Size     First Load JS
┌ ○ /                                      20.4 kB        93.5 kB
├   └ css/77e878d72d98d937.css             1.86 kB
├   /_app                                  0 B            73.1 kB
├ ○ /404                                   181 B          73.3 kB
└ λ /api/hello                             0 B            73.1 kB
+ First Load JS shared by all              75.9 kB
  ├ chunks/framework-2c79e2a64abdb08b.js   45.2 kB
  ├ chunks/main-f11614d8aa7ee555.js        26.8 kB
  ├ chunks/pages/_app-2cd5058a061812ad.js  307 B
  ├ chunks/webpack-8fa1640cc84ba8fe.js     750 B
  └ css/41d6bc5ddf0de1e4.css               2.77 kB

λ  (Server)  server-side renders at runtime (uses getInitialProps or getServerSideProps)
○  (Static)  automatically rendered as static HTML (uses no initial props)
```
- ✅ Fixed the problem of closing the CommandPalette by clicking on the sides - [Check](https://github.com/pheralb/superkey/blob/main/packages/component/src/components/command.tsx#L23).

Thanks for the report 🚀🚀
